drivers/bus/ancillary.c

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
@dmertman /* */ to be consistent with ancillary_bus.h

Reply:
@paravmellanox this is how we have it all our C files with // and /* in the headers. 

Our CI has already run checkpatch and there are no issues.

Reply:
except for "co-authored". I've seen "co-developed by".
maybe that'll fix it?

Reply:
> 
> 
> @paravmellanox this is how we have it all our C files with // and /* in the headers.
> 
> Our CI has already run checkpatch and there are no issues.

@ranj063 ok. great.
I will be sending internally soon for Jason G to review.